frontend/src/index.css

```diff
@@ -65,6 +65,52 @@ a {
   color: #1d7bd7;
 }
 
-.navlink {
-  color: inherit;
+
```

Comment:
Why have you removed the `navlink` class from there? 🤔 
It was responsible for links color in the Header component. They are not blue and not the dark grey color.
![Screenshot 2024-12-23 at 3 01 05 PM](https://github.com/user-attachments/assets/a7d80906-dd72-4ad3-89b6-30ae7c1deeac)